--- a/Gemfile   2013-04-05 11:51:22.361386708 +0100
+++ b/Gemfile   2013-04-05 11:51:52.992992856 +0100
@@ -10,6 +10,9 @@

 gem "rails", "3.2.13"

+# Use patched charlock_holmes with linking against system libmagic library.
+gem "charlock_holmes", "0.6.9", :git => "https://github.com/Danie/charlock_holmes.git", :branch => "gitlab_patch"
+
 # Supported DBs
 gem "mysql2", group: :mysql
 gem "pg", group: :postgres
